logging/OpenShift.md

```diff
@@ -111,6 +111,21 @@ oc -n $LOG_NS delete route v4m-es-client-service
 ***Note:*** Do not use the `es_nodeport_enable.sh` and `es_nodeport_enable.sh` scripts 
 in an OpenShift environment.  
 
+#### Customizing Routes
+
+Routes may be configured to be host-based (default) or path-based. When
+using host-based routes, access to logging applications will be by hostname
+only - for example: `https://v4m-kibana-monitoring.apps.my-openshift-cluster.com`.
+Path-based routes will result in a url ending in `/appName` - for example:
+`https://v4m-logging.apps.my-openshift-cluster.com/kibana`.
+
+Use `OPENSHIFT_PATH_ROUTES=true` in `$USER_DIR/user.env` to use path-based
+routes.
+
```

Comment:
They could define this env var in the $USER_DIR/logging/user.env file instead of the top-level user.env file too; correct?  If so, that would limit its impact to the logging applications only.  Should we mention that option?  I'm reluctant to document every possible combination, but given that this document is in the logging sub-directory, maybe we need to touch on it.

Reply:
It's our standard customization...seems unnecessary to re-hash how `user.env` files works in a `$USER_DIR`. I'll defer to @davidbass13 .

Reply:
I think it wouldn't hurt to mention it. I'd word it as 'Specify 'OPENSHIFT_PATH_ROUTES=true' in the '$USER_DIR/user.env' file (for both logging and monitoring) or the '$USER_DIR/logging/user.env' file (for only logging components) to use path-based routes.'